server/src/concepts/user/model/User.js

```diff
@@ -0,0 +1,44 @@
+import mongoose from "mongoose";
+
+const userSchema = new mongoose.Schema({
+  _id: mongoose.Schema.Types.ObjectId,
+  email: {
+    type: String,
+    required: true,
+    min: 6,
+    max: 255,
+    unique: true,
+    match:
+      /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/,
```

Comment:
I think it would be better to create a `const EMAIL_REGEX=/[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/` above `const userSchema`
This is not wrong but when there is some constant or some long line it is better to do it like this is kind a good practice.

Reply:
Ok, thanks for the advice!